TestsFoundation/TestsFoundation/UITests/CoreUITestCase.swift

```diff
@@ -132,7 +132,7 @@ open class CoreUITestCase: XCTestCase {
         // and also write the request/response in plain text to the log file
         case passThruAndLog(toPath: String)
     }
-    open var missingMockBehavior: MissingMockBehavior = .failTest
+    open var missingMockBehavior: MissingMockBehavior = .allow
```

Comment:
Since we aren't writing too many new tests with this mock framework, I'm not very worried about missing mocks anymore, and this will hopefully make UI tests less flaky.